source/math/FloatComplexVectorList.ooc

```diff
@@ -161,4 +174,32 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 			result add(value)
 		result
 	}
+	_fastFourierTransformBufferSize: static func (inputSize: Int) -> Int {
+		logValue := inputSize as Float log() / 2.0f log()
```

Comment:
Alright, you can merge this. I will take a look at updating the *Math* tomorrow.